end-to-end.test.js

```diff
@@ -45,6 +60,24 @@ describe("End to End", () => {
     await expect(page).toMatch("Go to the list of Articles");
     await expect(page).toMatch("Go to the list of Talks");
     await expect(page).toMatch("Go to the list of Shared Links");
+    const ogURLContent = await page.$eval(
```

Comment:
You can extract this to a function outside the test and reuse it passing the page

Reply:
For only the part of getting the content of the meta tag?

Reply:
```javascript
async function getMetaTagContent(page, property) {
  return await page.$eval(
    `meta[property='${property}']`,
    (element) => element.content
  );
}
```